curriculum/js1/9.js

```diff
@@ -4,8 +4,12 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a=0) => {
+  if(a%7===0)
```

Comment:
solution(8) would return 21 right? did you run tests?

Reply:
Yes I ran tests and all 4 of them passed